### PR TITLE
Feature/Add script phase to cocoapods

### PR DESCRIPTION
**Pull request by alyakan**
## TODO
- [ ] Turn `echo` into `warning`
- [ ] Check if we're in `Pods` directory before `cd`

### CODE REVIEW

File: Instabug.podspec [hunk 1]
```diff
@@ -11,6 +11,16 @@ Pod::Spec.new do |s|
   s.frameworks   = 'AVFoundation', 'CoreGraphics', 'CoreMotion', 'SystemConfiguration', 'CoreTelephony', 'UIKit', 'CoreMedia', 'CoreVideo', 'CoreData'
   s.xcconfig     =  { 'FRAMEWORK_SEARCH_PATHS' => '"$(PODS_ROOT)/Instabug/"' }
   s.requires_arc = true
+  s.script_phase = {
+    :name => 'Upload dSYMs',
+    :script => 'SCRIPT_SRC=$(find "$PROJECT_DIR" -name "Instabug_dsym_upload.sh" | head -1)
+    if [ ! "${SCRIPT_SRC}" ]; then
+      echo "Instabug: err: script not found. Make sure that you are including Instabug.bundle in your project directory"
```
Comment (Kmohamed) [inline Instabug.podspec]: did you tried to print this error ?

**Comment by lovluv** [comment]
No

**Comment by keith** [comment]
I think people have to opt into this so it's probably ok but I wouldn't want to upload Dsyms every time I was doing a clean build.

**Comment by Kmohamed** [comment]
@keith I am sorry but why anyone wouldn't want to upload Dsyms every time  ?
And do you have any idea how can we make it optional ?

File: Instabug.podspec (hunk 1 above)
Comment (Kmohamed) [reply Instabug.podspec]: Thanks a lot 🙏

**Comment by alyakan** [comment]
@keith Currenlt the `script_phase` has no option for *Run only when installing* so this PR has been postponed until we either find another solution or hopefully Cocoapods would provide it.

**Pull request closed without merging**